src/test/resources/Admin.feature

```diff
@@ -0,0 +1,158 @@
+@Admin
+Feature:
+  As a user I want to be able to create a user in the admin form
+
+  Background:
+    Given user goes to "logInView" page
+
+  @3
+Scenario: login and access the administrator view to create a user
+Given I have a user with "ulises.santiago@gmail.com" email
+And password as "ulisesSantiago1"
+When user choose to sign in
+Then should be logged into application
+When goes to the webpage of Admin for create a user
```

Comment:
For this Admin feature Do you consider the posibility to add these five steps as part of the background? I notice you need to call them  for all the scenarios on this feature file.